torchgeo/models/resnet.py

```diff
@@ -108,59 +96,47 @@ class ResNet50_Weights(WeightsEnum):  # type: ignore[misc]
         },
     )
 
-    SENTINEL2_ALL_MOCO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_all_moco/"
-            "resolve/main/resnet50_sentinel2_all_moco.pth"
-        ),
+    SENTINEL2_ALL_DINO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_all_dino/resolve/main/resnet50_sentinel2_all_dino-d6c330e9.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
             "in_chans": 13,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
-            "ssl_method": "moco",
+            "ssl_method": "dino",
         },
     )
 
-    SENTINEL2_RGB_MOCO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_rgb_moco/"
-            "resolve/main/resnet50_sentinel2_rgb_moco.pth"
-        ),
+    SENTINEL2_ALL_MOCO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_all_moco/resolve/main/resnet50_sentinel2_all_moco-df8b932e.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
-            "in_chans": 3,
+            "in_chans": 13,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
             "ssl_method": "moco",
         },
     )
 
-    SENTINEL2_ALL_DINO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_all_dino/"
-            "resolve/main/resnet50_sentinel2_all_dino.pth"
-        ),
+    SENTINEL2_RGB_MOCO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_rgb_moco/resolve/main/resnet50_sentinel2_rgb_moco-2b57ba8b.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
-            "in_chans": 13,
+            "in_chans": 3,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
-            "ssl_method": "dino",
+            "ssl_method": "moco",
         },
     )
 
     SENTINEL2_RGB_SECO = Weights(
```

Comment:
This weight has a problem but I can't figure out what it is:
```console
$ pytest -m slow tests/models/test_resnet.py::TestResNet50::test_resnet_download[ResNet50_Weights.SENTINEL2_RGB_SECO]
...
_pickle.UnpicklingError: invalid load key, '<'.
```
The error doesn't occur with the previous weight file, and both the old and new weight files have the same checksum. We'll need to figure out what's going on here before this PR can be merged.